Minor line editing, one query (#171)
WebOfTrustInfo/rwot9-prague: draft-documents/encrypted-data-vaults.md

Identity Hubs uses JSON Web Tokens (JWTs) and specified endpoints. Multiple requests
are required, first to retrieve metadata for the desired data object(s), and then
to retrieve the sequence of commits that make up the actual data. Mechanisms for
authentication are still under development. Access control is carried out via
posting to the Permissions interface.

Tahoe-LAFS uses a client-gateway-storage server architecture, whereby a client
passes data to a gateway server for encryption and chunking. The gateway, in

> turn, stores the individual chunks on a cluster of storage servers. (Several

Reviewer: ```suggestion
turn, stores the individual chunks on a cluster of storage servers. Several
```

> copies of the data are stored, for greater availability and disaster recovery.)

Reviewer: ```suggestion
copies of the data are stored, for greater availability and disaster recovery.
```

over their data without requiring centralized authorities or proprietary
technologies.

> Requiring client-side (edge) encryption for all data and metadata — enabling the

Reviewer: ```suggestion
Requiring client-side (edge) encryption for all data and metadata at the same time as enabling the
```

> user to store data on multiple devices and to share data with others, while having 

Reviewer: ```suggestion
user to store data on multiple devices and to share data with others, whilst also having 
```

> searchable or queryable data — has been historically very difficult to

Reviewer: ```suggestion
searchable or queryable data, has been historically very difficult to
```

implement in one system. We can see from this survey that trade-offs are often
made which sacrifice privacy in favor of usability, or vice versa.

Due to a number of maturing technologies and standards, we are hopeful that such
trade-offs are no longer necessary, and that it is possible to design a
privacy-preserving protocol for encrypted decentralized data storage that has
broad practical appeal.

## Core Use Cases

> The following four use cases have been identified as representative of common

Reviewer: :woman_facepalming: :+1:

- Choice of authorization strategy
- Choice of change control / conflict resolution strategy
- Notification / pub-sub mechanisms
- With respect to the authorization model, does the vault merely enforce authorization rules or act as an authorization server?

> - What are the assumptions of trust the host the vault must provide, enforcement of authorization rules? [[THIS NEEDS CLEANUP/CLARIFICATION]] Potential attack vectors from the host of the vault?

Reviewer: ```suggestion
- How can end users be reassured of the trustworthiness of a vault host?
- Further analysis of potential attack vectors of malicious servers, and mitigation techniques.
```
